app/views/applicants/unassigned.html.erb

```diff
@@ -1,27 +1,56 @@
+<script src="https://cdn.jsdelivr.net/npm/datalist-polyfill@1.22.1/datalist-polyfill.min.js"></script>
+
+<datalist id="crafters-list-Chicago">
+  <option value="Auto Assign"></option>
+  <% @chicago_crafters.each do |crafter| %>
+    <option value="<%= crafter.name %>"><%= crafter.name %></option>
+  <% end %>
+</datalist>
+
+<datalist id="crafters-list-London">
+  <option value="Auto Assign"></option>
+  <% @london_crafters.each do |crafter| %>
```

Comment:
Again there's probably a better way to define these datalists without having to define a datalist for each individual location.